components/__tests__/fixtures/content/es/foo.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  foo: 'foo in español',
+  bar: '',
+  baz: null,
+  sections: [require('./bar')],
```

Comment:
Yeah, so I just want to make sure that when the `sections` are in a different order, we use the right fallback values. So alternatively, you could take the english version and change it to: `sections: [require('./baz'), require('./bar')]` just so we're sure the fallbacks apply to the right translations :+1: